packages/react-components/react-provider/src/components/FluentProvider/FluentProvider.types.ts

```diff
@@ -1,4 +1,4 @@
-import type { IconDirectionContextValue } from '@fluentui/react-icons';
+import type { IconDirectionContextValue } from '@fluentui/react-icons/providers';
```

Comment:
based on offline convo I assume this tries to use package.json export map mapping? 

if yes, this will explode in our repo ( we would need to enable nodeResolution `bundler` within TS, which might come with it's own issues cross fluent versions, but most importantly in user land if it becomes part of our public api ( shipped within `index.d.ts` ) and users don't have `skipLibCheck: false` within their TS configs ( which they should in application land ) but we cannot be sure, thus this needs to follow same pattern as you did with the `/lib/` nested ( private path ) below.